app/integrations/lib/rocketchat.js

```diff
@@ -1,4 +1,8 @@
+import { RocketChatIntegrationHandler } from '../server/lib/triggerHandler';
```

Comment:
this file is imported by the client bundle, so doing this doesn't work.. 

even if this worked, it would fail because `../server/lib/triggerHandler` has a dependency on Node's `vm` , which would fail when trying to use on client.